cmd/state/internal/cmdtree/branch.go

```diff
@@ -5,6 +5,7 @@ import (
 	"github.com/ActiveState/cli/internal/locale"
 	"github.com/ActiveState/cli/internal/primer"
 	"github.com/ActiveState/cli/internal/runners/branch"
+	"github.com/ActiveState/cli/internal/runners/swtch"
```

Comment:
You're using 'switch' everywhere else, so why 'swtch'? There doesn't appear to be a name collision as far as I can see...

Reply:
`switch` is a keyword in go so you cannot name a package `switch`

Reply:
lol, oops